studia-historica-upsaliensa.csl

```diff
@@ -0,0 +1,588 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="note" version="1.0" and="symbol" initialize="false" demote-non-dropping-particle="never" xmlns="http://purl.org/net/xbiblio/csl">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Studia Historica Upsaliensa </title>
```

Comment:
We name all our university department styles as `<title>name of university - name of department</title>`. Would `<title>Uppsala universitet - Historiska institutionen</title>` be acceptable? (if so, the style's file name, style ID, and "self" link would have to be updated to match)